### ISSUE TITLE
Volume blocklet causing slow shutdown times

### ISSUE THREAD

**Issue by jasper1378** [jasper1378/i3neostatus#38]
I know this seems really weird, but bear with me. I was noticing that both my thinkpad and my desktop were experiencing slow shutdown times. I finally got around to taking a look at the systemd journal and found this:
```
Jan 01 21:32:02 jasper-t14gen3 systemd[1]: session-2.scope: Stopping timed out. Killing.
Jan 01 21:32:02 jasper-t14gen3 systemd[1]: session-2.scope: Killing process 1222 (volume) with signal SIG>
Jan 01 21:32:02 jasper-t14gen3 systemd[1]: session-2.scope: Killing process 1230 (threaded-ml) with signa>
Jan 01 21:32:02 jasper-t14gen3 systemd[1]: session-2.scope: Failed with result 'timeout'.
```

So to test my theory I tried running `killall volume` before running `poweroff` and sure enough the shutdown time was vastly reduced.

Perhaps it is something to do with how the blocklet asynchronously connects to the pulseaudio server; or how the signal handlers are set up?

Right now the volume blocklet makes use of some third-party code from i3status to read volume data, perhaps this is a good opportunity to rewrite the blocklet to use our own code?

**Comment by jasper1378** [comment]
Reminder to myself to update the documentation after rewrite.

**Comment by jasper1378** [comment]
Closed by #44.
